generators/app/templates/src/config/navigation.ejs

```diff
@@ -0,0 +1,77 @@
+<%_ if (features.drawerandroid || features.drawerios) { _%>
+import React from 'react';
+import { Button } from 'react-native';
+
+<%_ } _%>
+import * as Routes from '../constants/routes';
+import { blue, white } from '../constants/colors';
+<%_ if (features.drawerandroid || features.drawerios) { _%>
+import { actionCreators as drawerActions } from '../redux/drawer/actions';
+<%_ } _%>
+
+import I18n from './i18n';
+
+// Default nav options for all screens
+const defaultNavOptions = ({ navigation }) => ({
+  // Change screen title from i18n traslates files
+  title: I18n.t(`app:${navigation.state.routeName}`),
+
+  // TODO: The following options are examples. Change them to your need
+  headerStyle: {
+    backgroundColor: blue
+  },
+  headerBackTitleStyle: {
+    color: white
+  },
+  headerTitleStyle: {
+    color: white
+    // TODO: If you want to change the header font, apply these options
+    // Replace the font with the need
+    // fontWeight: 'normal',
+    // fontFamily: 'Lato-Bold'
+  },
+  headerTintColor: white
+});
+
+export const screensNavOptions = {
+  // TODO: Add here the screens nav options that changes with respect to
+  // the default ones defined in defaultNavOptions, for example...
+  /*
+  [Routes.Home]: {
+    title: 'Home'
+  },
+  [Routes.Login]: {
+    header: null
+  }
+  */
+  <%_ if (features.login) { _%>
+  [Routes.InitialLoading]: {
+    header: null,
+    headerStyle: {
+      elevation: 0,
+      shadowOpacity: 0
```

Comment:
I'd name this configuration as
```
const headerPiola = {
  header: null,
  headerStyle: {
    elevation:0,
    shadowOpacity: 0
  }
}
```
or something else, so we can reuse these definitions in another routes / components. What do you think ?

Reply:
🤔 mmm i think that is particular for each project. I'd let the developer define that. Now I'd leave it that way because of the simple fact that the bootstrap is used only once